community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/spi/v3_1/TransactionBoundQueryContext.scala

```diff
@@ -136,12 +136,12 @@ final class TransactionBoundQueryContext(txContext: TransactionalContextWrapper)
     new BeansAPIRelationshipIterator(relationships, entityAccessor)
   }
 
-  override def indexSeek(index: IndexDescriptor, value: Any) = {
+  override def indexSeek(index: CypherIndexDescriptor, value: Any) = {
```

Comment:
Why all these changes? There is only one IndexDescriptor, so there is no ambiguity to resolve with the prefix.